llvm/test/TableGen/anonymous-location.td

```diff
@@ -0,0 +1,16 @@
+// RUN: llvm-tblgen --print-detailed-records %s | FileCheck %s -DFILE=%basename_s
+
+class A<int a> {
+  int Num = a;
+}
+
+// Verify that the location of the anonymous record instantiated
+// for A<10> and A<11> is correct. It should show the line where the
+// anonymous record was instantiated and not the line where the class
+// was defined.
+def y {
+  // CHECK: anonymous_0 |[[FILE]]:[[@LINE+1]]|
```

Comment:
I would just relax the filename check to be a regex accepting anything, and add the new lit feature in a follow up PR. If anything breaks a build somewhere, it will be this

Reply:
Done